plugins/thumbnail/lib/model/thumbnailFactory/kImageTransformationAdapter.php

```diff
@@ -101,12 +101,22 @@ protected function handleActionType($step)
 		switch($this->parameters->get(kThumbFactoryFieldName::TYPE))
 		{
 			case kExtwidgetThumbnailActionType::RESIZE:
+				if($this->parameters->get(kThumbFactoryFieldName::CROP_X) || $this->parameters->get(kThumbFactoryFieldName::CROP_Y) || $this->parameters->get(kThumbFactoryFieldName::CROP_WIDTH) || $this->parameters->get(kThumbFactoryFieldName::CROP_HEIGHT))
```

Comment:
Do really need to test all this cases since addInitialCrop protect itself anyway?

Reply:
Yes, to avoid unnecessary crop processing